src/resmom_win/start_exec.c

```diff
@@ -2787,18 +2797,23 @@ start_process(task *ptask, char **argv, char **envp)
 			(PLONG)NULL, FILE_END);
 	}
 	
+	/* If we fail to get cmd shell(unlikely), use "cmd.exe" as shell */
+	if (0 != get_cmd_shell(cmd_shell, sizeof(cmd_shell)))
```

Comment:
I have a habit of putting the constant first in a variable comparison since it compels the compiler to flag an error on accidental assignment = in place of ==(equal to). For consistency, I have adopted this for all comparisons. I can change it as you suggested.